<launch>

	<arg name="min"  default="150"/>
  	<arg name="max"  default="250"/>

	<node pkg= "line_nav" name="hough_image_nav" type="hough_image_nav.py"  output="screen" >
		<param name="min_edge" value="$(arg min)"/>
		<param name="max_edge" value="$(arg max)"/>
	</node>

	<node pkg= "line_nav" name="callRNN_direction" type="callRNN_direction.py"  output="screen" />

	<node pkg= "line_nav" name="img_RCNN_compressed" type="img_RCNN_compressed.py"  output="screen" />

	<node pkg= "line_nav" name="pose_RCNN" type="pose_RCNN.py"  output="screen" />

	<node pkg= "line_nav" name="fusion_rcnn_hough" type="fusion_rcnn_hough.py"  output="screen" />


	<!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find line_nav)/rviz/rviz_config_hough.rviz"/> -->
	<!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find line_nav)/rviz/rviz_config_rcnn.rviz"/> -->
	
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find line_nav)/rviz/rviz_config_all.rviz"/>

</launch>
